File: 05 - Python/04-03-00-OperadorTernario-ULP.py
# Sentencia if-else en Python
mi_numero = int(input('Proporciona un numero: '))
# Revisamos si el numero es mayor que cero
if mi_numero > 0:
    print(f'Valor positivo: {mi_numero}')
else:
    print(f'Valor cero o negativo: {mi_numero}')

# Operador ternario
print()
print('Positivo') if mi_numero > 0 else print('Cero o Negativo')

# Sentencia if vacia
if mi_numero > 0:
    pass

File: 06 - Java/04-03-00-OperadorTernario-ULP.java
import java.util.Scanner;

public class ProgramaPrincipal {

    public static void main(String args[]){
        // Sentencia if-else
        // Pedimos un valor
        System.out.print("Proporciona un numero: ");
        var numero = Integer.parseInt(new Scanner(System.in).nextLine());
        // Verificamos si es un valor positivo
//        if (numero > 0){
//            System.out.println("Valor positivo: " + numero);
//        }
//        else{
//            System.out.println("Valor Cero o Negativo: " + numero);
//        }
        // Operador Ternario (regresa un valor)
        var resultado = (numero > 0) ? "Positivo" : "Cero o Negativo";
        System.out.println("Resultado: " + resultado);
    }
}
